free@free:~$ python auth_synch_failure_attack.py 
retrieving emm_ctx: 0.000445127487183
lib_nas_msg_decode time : 5.22136688232e-05
sending true
0completed
1completed
2completed
3completed
4completed
^CTraceback (most recent call last):
  File "auth_synch_failure_attack.py", line 23, in <module>
    message_id, ue_id, length, msg = le.recv(buf_size)
  File "/home/free/leap/func.py", line 79, in recv
    fromaddr, flags, buf, notif = self.s.sctp_recv(size)
  File "/home/free/leap/sctp.py", line 1239, in sctp_recv
KeyboardInterrupt
